### PR TITLE
Map picker view tweaks

### PR DESCRIPTION
**Pull request by ThislsAUsername**

### CODE REVIEW

**Comment by ThatIsAUsername** [comment]
Kinda enjoyed the pixely map. This is probs better though.

Do we also want to scale it in between the 1x -> 2x (and beyond) sizes?

**Comment by ThislsAUsername** [comment]
Not really? I like my exact pixels.

**Comment by ThatIsAUsername** [comment]
On large maps (e.g. Europe) the white boxes in the team-select screen are not placed correctly.

File: src/UI/Art/SpriteArtist/MapSelectMenuArtist.java [hunk 1]
```diff
@@ -108,7 +116,7 @@ private static void drawMapSelectMenu(Graphics g, MapSelectController gameSetup)
     int maxMiniMapHeight = drawableHeight - MapInfoPaneDrawHeight;
 
     // Find the center of the minimap display.
-    int miniMapCenterX = nameSectionDrawWidth + (maxMiniMapWidth / 2);
+    int miniMapCenterX = nameSectionDrawWidth + ((maxMiniMapWidth+1) / 2);
```
Comment (ThatIsAUsername) [inline src/UI/Art/SpriteArtist/MapSelectMenuArtist.java:119]: Why +1? Is half the map width not the center of the map frame for some reason?

Comment (ThislsAUsername) [reply src/UI/Art/SpriteArtist/MapSelectMenuArtist.java:119]: I think it had a chance (with odd widths?) of overlapping the border between the two sections at certain sizes.
So I just said "no".

**Comment by ThislsAUsername** [comment]
> On large maps (e.g. Europe) the white boxes in the team-select screen are not placed correctly.

Good catch. Very amusing that the code was "right" aside from making the assumption that all scales are integer.

File: src/UI/Art/SpriteArtist/MapSelectMenuArtist.java (hunk 1 above)
Comment (ThatIsAUsername) [reply src/UI/Art/SpriteArtist/MapSelectMenuArtist.java:119]: Strange. Makes me think we must be off on some of the math somewhere, but maybe not worth hunting down.

**Pull request merged**